ts/store/reducers/__mock__/backendStatus.ts

```diff
@@ -128,36 +130,44 @@ export const baseRawBackendStatus: BackendStatus = {
     },
     bancomatpay: {
       is_visible: false,
-      level: "warning",
+      level: LevelEnum.warning,
       message: {
         "it-IT": "bancomatpay",
         "en-EN": "bancomatpay"
       }
     },
     digital_payments: {
       is_visible: false,
-      level: "warning",
+      level: LevelEnum.warning,
       message: {
         "it-IT": "digital_payments",
         "en-EN": "digital_payments"
       }
     },
     cobadge: {
       is_visible: false,
-      level: "warning",
+      level: LevelEnum.warning,
       message: {
         "it-IT": "cobadge",
         "en-EN": "cobadge"
       }
     },
     euCovidCert: {
       is_visible: false,
-      level: "warning",
```

Comment:
There are a few other `level: "warning"` in [ts/store/reducers/__tests__/backendStatus.test.ts](https://github.com/pagopa/io-app/pull/3221/files#diff-1f1ef98955c8528b761adad8533a0ac2e15e4d90285cf44ab2ae9b048b7cad0eR82) that we shall update.

Reply:
I just checked. Are you sure to have the last commit of this branches?
That file should be right sync with new spec
https://github.com/pagopa/io-app/blob/aa8987e365f08318d140630531aea62645ce96a3/ts/store/reducers/__mock__/backendStatus.ts
Otherwise I guess we should had a complire error 

Could you confirm?

Reply:
Just checked with the latest commit: I can see many examples of `level: "warning"` in `backendStatus.test.ts`.
It works fine and runs the tests correctly, though since you're after this task you might want to replace all of them with the new enum.

Reply:
you are totally right.
I didn't find what you mean.
This commit should resolve (I also found a point where this field is compared incorrectly)
https://github.com/pagopa/io-app/pull/3221/commits/1dc9f3d8c2ff0349f197322fc019ba0b917091d8